src/odemis/acq/align/spot.py

```diff
@@ -288,7 +288,7 @@ def FindSpot(image, sensitivity_limit=100):
             LookupError() if spot was not found
     """
     subimages, subimage_coordinates = coordinates.DivideInNeighborhoods(image, (1, 1), 20, sensitivity_limit)
-    if subimages == []:
+    if not subimages:
```

Comment:
Note that this is *not* equivalent. For instance, if `subimages` can be either `None` or `[]`, then it changes the behaviour of the code. That said, it'd be a pretty bad function signature. I'm pretty sure that in this case, it's fine to use `not subimages`, indeed.

Reply:
Would be checking `len(subimages)` be a better solution?

Reply:
`len(None)` raises an exception, so it wouldn't be better, if the function returned None. Anyway, I'm pretty sure it never returns None, so the code is fine. It was just a note, just to make sure you're aware of the semantic change.

Reply:
Think I got it now :)